lib/tasks/deployment/20250603135846_group_admins_with_cluster_schools.rake

```diff
@@ -0,0 +1,62 @@
+# rubocop:disable Metrics/BlockLength
+namespace :after_party do
+  desc 'Deployment task: group_admins_with_cluster_schools'
+  task group_admins_with_cluster_schools: :environment do
+    puts "Running deploy task 'group_admins_with_cluster_schools'"
+
+    # Find group admins who have also been manually linked to schools, e.g. were
+    # originally school admins. Remove all those associations as they're unnecessary
```

Comment:
On my local machine I found a user (Jasmine Newhouse) with a cluster school in a different group to her school group, might not be the same on live though. Assume we are ok to remove cluster school relationships in these cases, as it shouldn't be possible for a user to be a group_admin and also have a cluster school in a different group?

Reply:
This is unusual but in practice I don't think it'd be doing anything. As a group user they should be getting group admin specific permissions so I don't think that relationship would do anything. I'll take another look though.

Reply:
I've confirmed that, when this user logs in they get a group admin view and don't have school admin privileges for the other school in their list. However they are linked directly to a school which is impacting what page they see when they sign on, so will update the migration to handle that too.